### DISCUSSION TITLE
Error in transaction.d.ts

### DISCUSSION THREAD

**Discussion by asifhameed8** [wevm/viem#751] [Question]
Hello i facing error in transaction.d.ts

![image](https://github.com/wagmi-dev/viem/assets/73539971/9b38b570-1e98-42fd-9808-412592d9e9e3)

I am using Angular 16 with typescript 5+. Urgent help would be appreciated.

**Comment by jxom** [comment]
Got some code of the usage we can look at? tsconfig would be handy too.

**Reply by asifhameed8** [reply]
Sure here is the tsconfig.

/* To learn more about this file see: https://angular.io/config/tsconfig. */
{
  "compileOnSave": false,
  "compilerOptions": {
    "allowSyntheticDefaultImports": true,
    "baseUrl": "src",
    "outDir": "./dist/out-tsc",
    "forceConsistentCasingInFileNames": true,
    "strict": true,
    "noImplicitOverride": true,
    "noPropertyAccessFromIndexSignature": true,
    "noImplicitReturns": true,
    "noFallthroughCasesInSwitch": true,
    "strictPropertyInitialization": false,
    "sourceMap": true,
    "declaration": false,
    "downlevelIteration": true,
    "experimentalDecorators": true,
    "moduleResolution": "node",
    "importHelpers": true,
    "target": "ES2022",
    "module": "es2020",
    "lib": ["es2020", "dom"],
    "useDefineForClassFields": false,
    "useUnknownInCatchVariables": false,

    "paths": {
      "@shared/*": ["app/shared/*"],
      "@directive/*": ["app/directives/*"],
      "@service/*": ["app/services/*"],
      "@pipe/*": ["app/pipes/*"],
      "@component/*": ["app/components/*"],
      "@env/*": ["environments/*"],
      "@utils/*": ["app/utils/*"]
    },
    "types": ["@angular/localize", "jasmine"],

    "emitDecoratorMetadata": false
  },
  "angularCompilerOptions": {
    "enableI18nLegacyMessageIdFormat": false,
    "strictInjectionParameters": true,
    "strictInputAccessModifiers": true,
    "strictTemplates": true
  }
}

**Reply by asifhameed8** [reply]
There is no usage yet. I just installed using (npm i viem). And after compiling this error comes.

**Reply by jxom** [reply]
What if you set `skipLibCheck` to true?

**Reply by asifhameed8** [reply]
It suppresses the error. But i was looking for some solution. What if it breaks during runtime.
